src/pages/initiial/index.js

```diff
@@ -0,0 +1,73 @@
+import { PrimaryButton } from '@fluentui/react';
```

Comment:
/minor folder name initiial here

Reply:
Sorry, I am not sure what do you mean by this?

Reply:
Oh. The typo in the folder name. You are right!